### ISSUE TITLE
Make all the pages related to teachers responsive 

### ISSUE THREAD

**Issue by coding-geek21** [coding-geek21/classDeck#65]
## Describe the bug

![image](https://user-images.githubusercontent.com/53329034/158091263-46461ab5-0ef2-44ee-a869-8f695f88b821.png)

All pages related to teachers look unresponsive in mobile view 

## Steps to Reproduce

Files to be made responsive :

- All files inside `classroom/templates/classroom/teachers` folder except teachers_home.html

 
## Expected Behavior

Make it responsive

**Comment by nitishsaini706** [comment]
@coding-geek21 I would like to work on this issue

**Comment by coding-geek21** [comment]
@nitishsaini706 any Updates?

**Comment by nitishsaini706** [comment]
@coding-geek21 yes, I've fixed the issue but when I reduce screen size to a little smaller width I, it's causing same issue, I'll fix it by today and raise a pr

**Comment by nitishsaini706** [comment]
@coding-geek21 I need your help, ping me on discord , I cannot continue without your help

**Comment by coding-geek21** [comment]
@nitishsaini706 Ping me on discord or in our project channel. I will help you

**Comment by nitishsaini706** [comment]
@coding-geek21 I've been trying to contact since last week

**Comment by coding-geek21** [comment]
> @coding-geek21 I've been trying to contact since last week

Ok what is your discord username?

**Comment by nitishsaini706** [comment]
![Uploading 16479457777121332386460901577.jpg…]()

**Comment by coding-geek21** [comment]
@nitishsaini706 any updates?

**Comment by nitishsaini706** [comment]
I can't install the project coz of dependencies problem, so I'm not able to work on it and no one helped me in groups so

**Comment by coding-geek21** [comment]
hey can you ping me in discord (username : Jayapritha N | PA | ClassDeck)
See here I couldn't see that image which you sent
![image](https://user-images.githubusercontent.com/53329034/163320210-c165759e-e8ee-46d9-864c-e4f10397e35f.png)

**Comment by coding-geek21** [comment]
@yaswanthsaivendra Can you help @nitishsaini706 ??

**Comment by nitishsaini706** [comment]
Will ping you after my classes

**Comment by yaswanthsaivendra** [comment]
Yeah sure!!

**Comment by coding-geek21** [comment]
@nitishsaini706 any updates?

**Comment by navyasharma0203** [comment]
If this issue isn't closed....can I work on this??

**Comment by jatinkr1405** [comment]
Currently my exams are going so I won't be able to do it.
